### ISSUE TITLE
eventResizableFromStart for timeGrid Layouts

### ISSUE THREAD

**Issue by wsergent** [fullcalendar/fullcalendar#4790]
Opening a new ticket as per this comment: https://github.com/fullcalendar/fullcalendar/issues/374#issuecomment-477319464

`eventResizableFromStart` only works on dayGrid views - it would be very useful to be able to use it in timeGrid views.

I understand the risk that two events starting/ending directly after each other could cause an issue (in that attempting to click on the start of one would be clicking on the end of the one before it) - but our use case is for a touch-enabled system, whereby we need to long-press on an event to bring up the resize option anyway - so in this case, there should be no issue with adjacent edges.

**Comment by arshaw** [comment]
I agree. It should be off by default, but if enabled, it should work in timeGrid views

**Comment by zsh1313** [comment]
It would be very handy if work in timeGrid view.

**Comment by matteorusso80** [comment]
I agree too

**Comment by Taaqif** [comment]
Any update on the possibility of this feature?

**Comment by Achuan-2** [comment]
+1
